Dockerfile

```diff
@@ -1,38 +1,17 @@
-FROM ubuntu:14.04
-MAINTAINER Colin Alston <colin@praekelt.com>
-RUN apt-get update && apt-get -y --force-yes install libjpeg-dev zlib1g-dev libxslt1-dev libpq-dev nginx redis-server supervisor python-dev python-pip
-RUN apt-get -y install libffi-dev gettext
-
-RUN pip install --upgrade pip
-
-ENV PROJECT_ROOT /deploy/
-ENV DJANGO_SETTINGS_MODULE tuneme.settings.docker
-
-WORKDIR /deploy/
-
-COPY tuneme /deploy/tuneme
-COPY polls /deploy/polls
-COPY locale /deploy/locale
-ADD manage.py /deploy/
-ADD requirements.txt /deploy/
-ADD setup.py /deploy/
-ADD README.rst /deploy/
-ADD VERSION /deploy/
-
+FROM praekeltfoundation/django-bootstrap
+COPY . /app
 RUN pip install -e .
 
-RUN mkdir -p /etc/supervisor/conf.d/
-RUN mkdir -p /var/log/supervisor
-RUN rm /etc/nginx/sites-enabled/default
-
-ADD docker/nginx.conf /etc/nginx/sites-enabled/molo.conf
-ADD docker/supervisor.conf /etc/supervisor/conf.d/molo.conf
-ADD docker/supervisord.conf /etc/supervisord.conf
-ADD docker/docker-start.sh /deploy/
-ADD docker/settings.py /deploy/tuneme/settings/docker.py
+RUN apt-get-install.sh gettext
 
-RUN chmod a+x /deploy/docker-start.sh
+ENV PROJECT_ROOT /app/ \
```

Comment:
I think these need to be in the `KEY=value` or `KEY="value"` form to support multiple per `ENV` command.